cdap-docs/developers-manual/source/building-blocks/schedules.rst

```diff
@@ -13,7 +13,7 @@ can add a schedule to a workflow using the ``scheduleWorkflow`` method of the
 `AbstractApplication class <../../reference-manual/javadocs/co/cask/cdap/api/app/AbstractApplication.html#scheduleWorkflow(co.cask.cdap.api.schedule.Schedule,%20java.lang.String)>`__
 
 The `Schedules <../../reference-manual/javadocs/co/cask/cdap/api/schedule/Schedules.html>`__
-class contains static methods to create schedules based on time, or schedules based on data availability.
+class contains a builder to create schedules based on time, or schedules based on data availability.
```

Comment:
yep, I was planning on doing that separately though, so that its not tied to these deprecation commits.